tests/gdx-tests/src/com/badlogic/gdx/tests/ClipboardTest.java

```diff
@@ -0,0 +1,61 @@
+package com.badlogic.gdx.tests;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.TextArea;
+import com.badlogic.gdx.scenes.scene2d.ui.TextButton;
+import com.badlogic.gdx.scenes.scene2d.utils.ChangeListener;
+import com.badlogic.gdx.tests.utils.GdxTest;
+
+public class ClipboardTest extends GdxTest {
+
+	Stage stage;
+	TextArea textArea;
+	TextButton buttonCopy;
+	TextButton buttonPaste;
+
+	@Override
+	public void create() {
+		stage = new Stage();
+		Skin skin = new Skin(Gdx.files.internal("data/uiskin.json"));
+		textArea = new TextArea("", skin);
+		buttonCopy = new TextButton("Copy", skin);
+		buttonPaste = new TextButton("Paste", skin);
+
+		textArea.setSize(Gdx.graphics.getWidth() / 3f, Gdx.graphics.getHeight() / 3f);

```

Comment:
Can we put magic numbers into appropriately named constants?

Reply:
Yes we could, but in my opinion it's unnecessary because it's only for the size of the textarea in the test

Reply:
For tests it makes sense. Thanks